src/libraries/Microsoft.PowerFx.Connectors/Execution/IConnectorInvoker.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    public class InvokerParameters
+    {
+        public QueryType QueryType { get; init; }
+
+        public HttpMethod HttpMethod { get; init; }
+
+        public string Url { get; init; }
+
+        public IReadOnlyDictionary<string, string> Headers { get; init; }
+
+        public string Body { get; init; }
+
+        public string ContentType { get; init; }        
+    }
+
+    public enum QueryType
```

Comment:
>QueryType [](http://example.com/codeflow?start=16&length=9)

Why do we need this at InvokeParameters?
Can these concerns already be abstracted away prior to creating the InvokerParameters?

Can this be removed?